_projects/forge.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Forge
+description: A generic low-code framework built on a config-driven tree walker.
+linkText: Learn more
```

Comment:
To meet accessibility requirements, can you select a phrase other than "Learn more" that is specific and descriptive?

Reply:
Updated to have similar syntax to other projects. Thanks.